api/src/main/java/io/opencensus/trace/propagation/PropagationComponent.java

```diff
@@ -49,6 +49,19 @@
   @ExperimentalApi
   public abstract TextFormat getB3Format();
 
+  /**
+   * Returns the B3 {@link TextFormat} with the provided implementations. See <a href=
+   * "https://github.com/openzipkin/b3-propagation">b3-propagation</a> for more information. If no
+   * implementation is provided then no-op implementation will be used. Extraction is equivalent to
```

Comment:
in general I think the fallback stuff and extract in general probably will end up needing to be decoupled. It sounds ok to always fall back to b3 multi at the moment, but you might imagine case in the future where someone will want "only b3 single" without having to wait for parse failures.. granted you only need to wait for 1 failure (failure to extract `X-B3-TraceId`

iotw in the future, with a way to register extract rules independently, we won't have to overload impls like this. I know it is doing this as the design is mostly copied from brave, but at any rate something to consider improving regardless even if not now.

Reply:
Would it make sense to add a `strict` flag? When it's set, fallbacks would be disabled.